API/UploadWordLinkRecording.php

```diff
@@ -20,7 +20,7 @@
 $androidId = trim($_POST['PhoneId']);
 $term = trim($_POST['term']);
 $audioRecordingFilename = htmlspecialchars(trim($_POST['audioRecordingFilename']));
```

Comment:
Can you do the same thing for audioRecordingFilename? Thanks!

Reply:
Done